.travis.yml

```diff
@@ -104,7 +104,7 @@ before_deploy:
 deploy:
   on:
     all_branches: true
-    condition: -v DEPLOY_S3_ACCESS_KEY && -v PRODUCT_TESTS
+    condition: -v DEPLOY_S3_ACCESS_KEY && -v UPLOAD_ARTIFACTS
```

Comment:
Let's just make this `MAVEN_CHECKS` instead of adding the extra variable (since this is the only place it's used after the above change).